django-stubs/db/models/fields/__init__.pyi

```diff
@@ -182,6 +183,7 @@ class Field(RegisterLookupMixin, Generic[_ST, _GT]):
         validators: Iterable[validators._ValidatorCallable] = ...,
         error_messages: _ErrorMessagesMapping | None = ...,
         db_comment: str | None = ...,
+        db_default: type[NOT_PROVIDED] | Expression | _ST = ...,
```

Comment:
Sorry for sitting on this for so long.

When I was thinking about this later, I realized that `_ST` typevar is usually unbound in model declarations (e.g. `field = CharField(db_default=...)` rather than `field = CharField[str, str](db_default=...)`; the typevars are resolved later by mypy plugin.

In mypy, this doesn't cause errors at usage location because unbound typevars default to `Any` in mypy. But I wasn't sure about pyright.

I tested this now, seems that pyright handles it the same way: https://pyright-play.net/?pyrightVersion=1.1.353&pythonVersion=3.12&pythonPlatform=Linux&strict=true&locale=en&code=GYJw9gtgBALgngBwJYDsDmUkQWEMoDiApikSEgMYA0UAKokQGoCGIAUG7VALx0MsgAFACJawgJQcKAG2YBnOVABCsgBYAxJEWkATQcVLkKAbVoBdcQC42UW1B1FgUAPrPUSGK8FztwGjoAjZwdgZgBXaRhLOigAHygAOTBSHigAOgzxKABaAD5E5KJojLSONmAwMFSVZg0tXUFA4MdwyO4ARgAmAGZJAGI7QaG7AD0x8YmJqBQqsnB2IA

I think we can improve this later with typevar defaults on Field classes. There are other reasons as well why that's useful. So I think it's OK to merge this as is.